app/services/user_alerts/alert_user_about_account_verified.rb

```diff
@@ -0,0 +1,16 @@
+module UserAlerts
+  class AlertUserAboutAccountVerified
+    def self.call(user, date_time, app, disavowal_token)
```

Comment:
can we make these keyword args so they're harder to mix up?

Reply:
yep, done in 8799437eb